src/plugins/advanced_settings/public/management_app/mount_management_section.tsx

```diff
@@ -47,6 +50,19 @@ const readOnlyBadge = {
   iconType: 'glasses',
 };
 
+// TODO: remove route `query` param in 7.12
```

Comment:
Why do you mean with this comment? We are on 7.12 😄

Reply:
Adding to @stratoula comment: what's the alternative?

Reply:
Yeah that's a little vague.

Currently the settings page looks for a route param (`settings/:query`) to auto populate the search query. This PR changes that to use a _query_ param instead (`settings?query=<query>`). But I figured it would be better to still allow the route param for one minor in case anyone is using it. I moved and rewored this comment to make that more clear.

See bff5739b9af93f8dafa391d7c08a98d73447a948

@lizozom does that answer your question too?

Reply:
I really don't think this use case deserves maintaining backwards compatibility.
At most, the user will lose their search param and will re-search on the page itself.
I think simplifying the code would be a nicer alternative.

What do you think?